code/modules/antagonists/wizard/equipment/spellbook_entries/offensive.dm

```diff
@@ -71,6 +71,13 @@
 	cost = 3
 	no_coexistance_typecache = list(/datum/action/cooldown/spell/conjure_item/infinite_guns/gun)
 
+/datum/spellbook_entry/FearAndHunger
+	name = "The Source of Fear and Hunger"
+	desc = "Call forth a spectral coin, casting it at your target while immobilizing them and forcing them to call heads or tails. If they all it incorrectly, they'll lose all their limbs."
+	spell_type = /datum/action/cooldown/spell/pointed/coinflip
+	category = "Offensive"
+	cost = 5 //5 because its a 50/50 chance aren't I so smart
```

Comment:
Why should anyone choose this over say, the delimbing spellblade?

Reply:
because it's funnier and you can't lose the spell, as opposed to losing the spell blade. Realistically that option is cheaper, however I think forcing someone into a dilemma situation where they have to rely on their luck rather than their skill is much more amusing.

Reply:
Plus it does provide immobization at it's current state- atleast until they pick their coin option, whereas someone could dodge the spell blade shots, or block them with a shield.